### DISCUSSION TITLE
[Suggestion] Windows version, configuration saves

### DISCUSSION THREAD

**Discussion by Stringendi** [nzp-team/nzportable#648] [General]
A fairly easy suggestion to implement is to add a confg so it saves resolution settings and everything else.

**Comment by MotoLegacy** [comment]
"Save & Apply" does this.

**Reply by Stringendi** [reply]
I can confirm it doesn't, must be a bug

**Reply by Stringendi** [reply]
I'm guessing it is where I have saved the game and doesn't have read permission

**Reply by Stringendi** [reply]
Yep that was the issue
